src/modules/popup.js

```diff
@@ -0,0 +1,110 @@
+    .then((response) => response.json())
+    .then((data) => {
+      data = data.meals;
+      mealData = data;
+      return fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/RKHWm1KUzqGmx0UP3nkk/comments?item_id=${IdMeal}`);
+    })
+    .then((response) => response.json())
+    .then((commData) => {
+      const div = `<div class="PopupBg" id="popup-1">
+      <section class="popup-detailCard">
+          <div class="prj-popup-header flexRow">
+              <h2 class="PopupTitle" id="popup-title">
+              ${mealData[0].strMeal}
+              </h2>
+              <i class="fas fa-times x-icon" id="x-icon-popup"></i>
+          </div>
+              <!-- Image and Details -->
+          <img src="${mealData[0].strMealThumb}" alt="Meal" class="popup-img" id="img-id">
+              <!-- @ take img with id -->
+          <div class="ProductPrice" id="product-price"> 
+              <p>${mealData[0].strTags}</p>
+              <p href="${mealData[0].strYoutube}">Youtube</p>
+              <p>${mealData[0].strIngredient1}</p>
+              <p>${mealData[0].strIngredient2}</p>
+          </div>
+      
+          <div class="ReviewTxtarea" id="review-txtarea">
+          <h3 id="review-count"></h3>    
+          <ul id="reviews-list"></ul>
+          </div>
+          <div>
+              <h3 id="give-review">Give Review</h3>
+              <form class="reviewForm" id="">
+                  <input id="form-name" type="text" placeholder="Your Name" required>
+                  <input id="form-review" type="text" placeholder="Give Review" required>
+                  <div class="style-popupbtn">
+                      <button type="button" class="button" id="submit-btn">
+                        Submit Review
+                      </button>
+                  </div>
+              </form>
+          </div>
+              <div id="counter"></div>
+            <p class="proj-primarytxt" id="proj-primarytxt-id"></p>
+            <div id="popuplist">
+              <ul class="no-bullets proj-list" id="tech-list-id">
+              </ul>
+              
+            </div>
+      </section>
+  </div>`;
+
+      overlay.innerHTML = div;
+      // Create DOM Elements for User Comments
+      const commentCount = document.getElementById('review-count');
+      let reviewCounts = commData.length;
+      if (!(reviewCounts > 0)) {
+        reviewCounts = 0;
+        commentCount.innerText = `Reviews (${reviewCounts})`;
+      } else {
+        commentCount.innerText = `Reviews  (${reviewCounts})`;
+        const Reviewul = document.getElementById('reviews-list');
+        commData.forEach((item) => {
+          const reviewDetail = document.createElement('li');
+          reviewDetail.innerText = `${item.creation_date} ${item.username}: ${item.comment}`;
+          reviewDetail.classList = 'review-detail';
+          Reviewul.append(reviewDetail);
+        });
+      }
```

Comment:
Kindly make sure that your project is using the same comment counter function that you are testing. Otherwise, you can not be sure if your comments counter is going to work properly. Please fix this. :pray: :pray: 

You are testing the function rewiewCounter function in your tests however you are not using that function to count the comments in your project. :eyes: :eyes: